packages/account/src/extension/ExtensionWeb2.ts

```diff
@@ -36,9 +36,8 @@ type AccountWithKeyPair = InjectedAccount & { keypair: KeyringPair };
  */
 export class ExtensionWeb2 extends Extension {
 	public async getAccount(
-		config: ProcaptchaClientConfigOutput,
 	): Promise<Account> {
-		const account = await this.createAccount(config);
+		const account = await this.createAccount();
```

Comment:
do we not need a config for this anymore?

Reply:
No we don't